.changeset/lucky-dancers-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": minor
+---
+
+Added a `styleType` prop to `<Select>`. The possible values are `'default'` (default prop value) and `'borderless'`.
```

Comment:
nit: This changeset could be reworded to center focus around "borderless". The "default" value is unimportant.

Reply:
Updated.